sparkle/CLI/validate_configured_vs_default.py

```diff
@@ -105,10 +107,21 @@ def main(argv: list[str]) -> None:
         configurator.output_path, solver, instance_set_train)
 
     if config_scenario is None:
-        print("No configuration scenario found for combination:\n"
-              f"{configurator.name} {solver.name} {instance_set_train.name}")
+        print("No configuration scenario found for:\n"
+              f"{configurator.name}: {solver.name} on {instance_set_train.name}")
         sys.exit(-1)
 
+    # Check if any jobs are still in the queue for this scenario
+    running_runs = jobs_help.get_runs_from_file(gv.settings().DEFAULT_log_output,
+                                                filter=[Status.WAITING, Status.RUNNING])
+    for run in running_runs:
+        # Bit of a hack to check if the job is still in the queue
```

Comment:
Would be nice if this eventually gets support from gunrunner to check this easily.